### ISSUE TITLE
[BUG] Issues running selenium tests with headless Firefox

### ISSUE THREAD

**Issue by cwperks** [opensearch-project/security-dashboards-plugin#1540]
**What is the bug?**

The PR to [fix the build on 2.x](https://github.com/opensearch-project/security-dashboards-plugin/pull/1533) is running into issues in the integration tests running selenium tests with Firefox.

The build output contains an error stating `InvalidArgumentError: binary is not a Firefox executable`.

See example Github Action with the failure here: https://github.com/opensearch-project/security-dashboards-plugin/actions/runs/5744153275/job/15572648806?pr=1533

**Comment by cwperks** [comment]
One thing that's odd is that there are progressively more `"` surrounding the debug output of `Browser path:`

```
console.debug
    Driver path: /opt/hostedtoolcache/geckodriver/0.33.0/x64/geckodriver

      at logOutput (plugins/security-dashboards-plugin/node_modules/selenium-webdriver/common/seleniumManager.js:142:15)
          at runMicrotasks (<anonymous>)

  console.debug
    Browser path: "/usr/lib/firefox/firefox.sh"

      at logOutput (plugins/security-dashboards-plugin/node_modules/selenium-webdriver/common/seleniumManager.js:142:15)
          at runMicrotasks (<anonymous>)

  console.debug
    Driver path: /opt/hostedtoolcache/geckodriver/0.33.0/x64/geckodriver

      at logOutput (plugins/security-dashboards-plugin/node_modules/selenium-webdriver/common/seleniumManager.js:142:15)
          at runMicrotasks (<anonymous>)

  console.debug
    Browser path: ""/usr/lib/firefox/firefox.sh""

      at logOutput (plugins/security-dashboards-plugin/node_modules/selenium-webdriver/common/seleniumManager.js:142:15)
          at runMicrotasks (<anonymous>)

  console.log
    Remove the Sample Data

      at Object.<anonymous> (plugins/security-dashboards-plugin/test/jest_integration/saml_auth.test.ts:178:13)
          at runMicrotasks (<anonymous>)

  console.error
    Unhandled error

      at process.uncaught (node_modules/jest-jasmine2/build/jasmine/Env.js:241:19)
      at process.emit (node_modules/source-map-support/source-map-support.js:516:21)

  console.error
    InvalidArgumentError: binary is not a Firefox executable
        at Object.throwDecodedError (/home/runner/work/security-dashboards-plugin/security-dashboards-plugin/OpenSearch-Dashboards/plugins/security-dashboards-plugin/node_modules/selenium-webdriver/lib/error.js:524:15)
        at parseHttpResponse (/home/runner/work/security-dashboards-plugin/security-dashboards-plugin/OpenSearch-Dashboards/plugins/security-dashboards-plugin/node_modules/selenium-webdriver/lib/http.js:601:13)
        at Executor.execute (/home/runner/work/security-dashboards-plugin/security-dashboards-plugin/OpenSearch-Dashboards/plugins/security-dashboards-plugin/node_modules/selenium-webdriver/lib/http.js:529:28)
        at runMicrotasks (<anonymous>)
        at processTicksAndRejections (node:internal/process/task_queues:96:5)

      at process.uncaught (node_modules/jest-jasmine2/build/jasmine/Env.js:242:19)
      at process.emit (node_modules/source-map-support/source-map-support.js:516:21)
```

**Comment by cwperks** [comment]
I have confirmed that this is an issue with selenium-webdriver 4.11.x, the build errors are resolved by forcing the resolution to 4.10.0
